nodepool/files/nodepool.yml

```diff
@@ -127,6 +127,19 @@ providers:
         availability-zones: []
         labels: *provider_pools_main_labels_block
 
+  - name: phobos-cloud
+    region-name: "RegionOne"
+    cloud: phobos-cloud
```

Comment:
Given the recent merge, this now needs to be ``phobos_nodepool`` to match up to the entry in clouds.yaml.